### PR TITLE
feat(opeanpi): Generate openapi scema dynamically on app startup.

### PR DESCRIPTION
**Pull request by samip5**

### CODE REVIEW

**Comment by romeq** [comment]
This version doesn't seem to compile on Docker, as there is no `java` installed, so I can not merge this as-is (the `master` branch cannot have anything that would crash an existing backend on automatic update).
 
I modified the Dockerfile to install `openjdk17` from Alpine Linux's repositories and it seemed to work fine. However I'm not certainly sure whether we should install java on the primary docker file; it increases the size of the production image a lot.

In other words, as much as I respect your will to contribute to this project I'm not going to merge this PR as long as the functionality is not separated from the actual API-server. 

Could the generator script be instead manually ran on development machines?

**Comment by samip5** [comment]
> Could the generator script be instead manually ran on development machines?

No, as the whole purpose of it is to dynamically use environment variables to match the deployment which shouldn't/cannot  be done anywhere else easily than runtime meaning that doing it in Docker's build stage is also not something that would satisfy that need.
